### PR TITLE
Bump WP and WC tested to versions

### PR DESCRIPTION
**Pull request by jrodger**
Bump the tested to versions for WordPress and WooCommerce.

WooCommerce won't output messages for plugins that don't declare support anymore, but since we're doing the WP one as well...

In addition to our usual CI tests (and the fact most of us have probably been developing against these version for a while) I've given everything a test with these latest versions. Everything looks good 👍

### CODE REVIEW

**Review by DanReyLop — APPROVED** [review]
:shipit: 

Side-note: Do you think we *really* need code review for these trivial PRs? I've used WC 4.8 but I'm still on WP 5.3, so my "approval" means basically "I take your word for it and I see you didn'y make a typo in the 2 characters you changed".

**Comment by jrodger** [comment]
> Side-note: Do you think we really need code review for these trivial PRs?

Yeah, we probably don't need it. I suppose even for trivial things we're doubling the chance of catching something with a second pair of eyes? Maybe WooCommerce started parsing these version strings differently or something? 🤷 

I wouldn't think twice about it if someone just merged something like this, and I would have done that if no one reviewed it before the release.

Now that we've both been flippant about it, let's see if the build breaks after the merge!

**Comment by DanReyLop** [comment]
> Now that we've both been flippant about it, let's see if the build breaks after the merge!

LOL

**Pull request merged**
